labels/de-DE.json

```diff
@@ -269,8 +269,8 @@
   "mobileOrderConfirmationMessage": "{0} Bestellbenachrichtigungen. Wir werden Ihnen eine SMS senden, wenn sich Ihr Bestellstatus ändert. Die Meldungsfrequenz kann variieren. Antworte {1}, um dich abzumelden. Es können Nachrichten- und Datentarife gelten",
   "mobileOrderOptInMessage": "Vielen Dank, dass Sie sich für SMS-Benachrichtigungen für Ihre Curbside-Bestellung angemeldet haben. Wir werden Sie benachrichtigen, wenn Ihre Bestellung zur Abholung bereit ist.",
   "mobilePickLocation": "Ramasser / Expédier de:",
-  "mobilePickupReminder":"Erinnerung - Artikel in Ihrer Bestellung {0} können abgeholt werden. Überprüfen Sie die Bestellung unter {1}",
-  "mobileReadyForPickup":"Artikel in Ihrer Bestellung {0} können um {1} abgeholt werden. Überprüfen Sie die Bestellung unter {2}",
+  "mobilePickupReminder": "Erinnerung - Artikel in Ihrer Bestellung {0} können abgeholt werden. Überprüfen Sie die Bestellung unter {1}",
```

Comment:
Don't reformat without reason. Makes it harder for our clients to merge changes. Please revert the whitespace changes.